dev/README_RELEASE_AIRFLOW.md

```diff
@@ -552,6 +599,59 @@ At this point we release an official package:
 
 - Update CHANGELOG.md with the details, and commit it.
 
+- Re-Tag & Push the constraints files with the final release version.
+
+    ```shell script
+    git checkout constraints-${RC}
+    git tag -s "constraints-${VERSION}"
+    git push origin "constraints-${VERSION}"
+    ```
+
+- Push Tag for the final version
+
+This step should only be done now and not before, because it triggers an automated build of
+the production docker image, using the packages that are currently released in PyPI
+(both airflow and latest provider packages).
+
+```shell script
+git push origin ${VERSION}
+```
```

Comment:
```suggestion
    This step should only be done now and not before, because it triggers an automated build of
    the production docker image, using the packages that are currently released in PyPI
    (both airflow and latest provider packages).

    ```shell script
    git push origin ${VERSION}
    ```
```